packages/dappmanager/src/calls/sensorsDataGet.ts

```diff
@@ -0,0 +1,8 @@
+import { getCpuTemperature } from "@dappnode/hostscriptsservices";
+
+/**
+ * Returns the sensors info of the host
+ */
+export async function sensorsDataGet(): Promise<number> {
```

Comment:
how confident we are about collecting the temperature from the host of the users? AFAIK we cannot ensure that every host machine from the user will have the paths/tools to collect the temperature, and in those cases, there will be shown an ugly error message.

Taking this into account I would consider this feature as optional since it could fail in some cases. So I would not throw the error if any, then the function could return `null` and the card will not be shown.

Something like 
```ts
/**
 * Returns the sensors info of the host
 */
export async function sensorsDataGet(): Promise<number | null> {
  try { 
  return await getCpuTemperature();
  } catch (e) {
  logs.warn(`Unable to get cpu temperature from the host: ${e}`)
  return null
  }
}

Reply:
Changed. Now if cannot get the temperature, it returns null. I also changed the frontend to not show the card if the function returns null